contents/handbook/small-teams/growth/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Growth Team
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+## People
+
+- [Emanuele Capparelli (Growth Engineer)](/handbook/company/team#emanuele-capparelli-growth-engineer)
+
+## Mission
+
+PostHog's company growth targets are met every quarter
+
+## Objective
+
+Each month, the Growth and [Exec team](/handbook/small-teams/exec) carry on a [Growth review session](/handbook/growth/growth-engineering/growth-sessions) to accelerate growth work, as well as define tasks for the Growth team.
+
+The Growth team focuses on Activation metrics, namely:
+
+- Signup activation rate: User has signup to the product -> User has successfully discovered the product's value
+- Card activation rate: User has inserted their card details
+- Paid activation rate: User has converted to a paying customer
+
+The team is primarily responsible for the following areas:
+
+- Signup flow
+- Onboarding and integrating PostHog with the customer's product
+- Billing and card activation flows
+- Messaging, emailing and notifications
+
+Current status of the Activation metrics can be checked on the [Growth dashboard](https://app.posthog.com/dashboard/54139) (internal only)
```

Comment:
This covers the areas that the growth team owns but doesn't seem to have specific objectives/targets of where we want to get to (more OKR style). I'd suggest we move this current section to an `Areas` section and then add objectives either in this PR or a separate PR

Reply:
For this quarter, I can just align the goal/OKR with the session recording's team of achieving 100k$ from new customers, then for next quarter we can think about if we want to have growth OKRs?

Reply:
sounds good to me